plugins/modules/firewalld.py

```diff
@@ -748,7 +748,7 @@ def main():
             interface=dict(type='str'),
             masquerade=dict(type='str'),
             offline=dict(type='bool'),
-            target=dict(type='str', choices=['default', 'ACCEPT', 'DROP', 'REJECT']),
+            target=dict(type='str', choices=['default', 'ACCEPT', 'DROP', '%%REJECT%%']),
```

Comment:
Add a test for this change in https://github.com/ansible-collections/ansible.posix/blob/main/tests/integration/targets/firewalld/tasks/zone_target_test_cases.yml

Reply:
added the tests